components/Carousel/Carousel.js

```diff
@@ -1,6 +1,66 @@
+// Your challenge is to write the functionality of a *Carousel* component. You have the HTML and CSS already in place.
```

Comment:
Awesome job of taking notes. I would suggest it might be better to put them in a separate file to keep this one looking neat. You can use a .md file to keep track of your notes and it will also be much easier for you to read.